src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -536,6 +536,7 @@ public PrivilegesEvaluatorResponse evaluate(PrivilegesEvaluationContext context)
                             if (replaceResult.accessDenied) {
                                 auditLog.logMissingPrivileges(action0, request, task);
                             } else {
+                                presponse.shouldSkipDlsValve = true;
```

Comment:
hmm, what happens without this?

Reply:
This is needed in the first instance that a private tenant is created. OpenSearch Dashboards will try to save a saved object to the private tenant index which is an index request. If the private tenant index does not yet exist it will follow-up by trying to auto create the index. These actions need to still be permitted.